primitives/trie/Cargo.toml

```diff
@@ -15,13 +15,13 @@ harness = false
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false }
 sp-std = { version = "2.0.0", default-features = false, path = "../std" }
 hash-db = { version = "0.15.2", default-features = false }
-trie-db = { version = "0.19.2", default-features = false }
-trie-root = { version = "0.15.2", default-features = false }
-memory-db = { version = "0.18.1", default-features = false }
+trie-db = { version = "0.20.0", default-features = false }
```

Comment:
Did you test this with Kusama? We already had trie updates breaking Kusama.

CC @cheme

Reply:
If I checked correctly this is just an update to parity-util-mem, so that should be fine.
But at the same time, testing with kusama don't take much time usually (I mean if there was actual change to trie I would try to resync from scratch which is a few hours (maybe a bit more)).

Reply:
I have seen that this also includes some prefixed key iterator?

Reply:
Oh yes. This iterator is not in use anywhere, it is new (would need to PR a change in substrate, and this will not be backward compatible regarding proof production (cumulus :eyes:).

Reply:
Okay :P On your responsibility :D

Reply:
There is only my bump of version with this update and a new iterator function, it's fine.

Reply:
https://github.com/paritytech/trie/pull/78
https://github.com/paritytech/trie/pull/79

Reply:
Oh my page didnt refresh and I answered to old stuff. Nevermind.